action/sealedenvelope.go

```diff
@@ -184,17 +184,11 @@ func wrapStakingActionIntoExecution(ab AbstractAction, toAddr []byte, pb proto.M
 	}, nil
 }
 
-// Verify verifies the action using sender's public key
-func (sealed *SealedEnvelope) Verify() error {
+// VerifyPubKey verifies the action using sender's public key
+func (sealed *SealedEnvelope) VerifyPubKey() error {
```

Comment:
not only the public key, but also signature. Because public key is needed to validate signature, `VerifySignature` may be a better name.

Reply:
IMO `Verify` is simple and clear enough

Reply:
I think `VerifySignature` is ok

Reply:
let's stick to `Verify`. IMO we should avoid keep changing func names, unless the name is very wrong? It may not be a big issue for each such change, but after 3~4 times refactor/change, the code can look quite differently, which IMO is not good for readability and grasp of the codebase